backend-nannyway/src/main/java/de/neuefische/nannyway/model/AddChildInCareDto.java

```diff
@@ -0,0 +1,17 @@
+package de.neuefische.nannyway.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class AddChildInCareDto {
+
+    private String firstName;
```

Comment:
you can use annotations to verify that the name ist not empty

Reply:
than you also have to add @Valid to the controller

Reply:
Done... what a struggle 😭